### PR TITLE
[#19] [iOS] [UI] As a user, I can see the log in screen

### LINKED ISSUE #19: [iOS] [UI] As a user, I can see the log in screen
# Why

To allow users to log in with their email and password, we must show the login screen to them.

# Acceptance Criteria

- Show the login screen with the following design

**Note**: We won't implement the forgot password feature; please don't add the `Forgot?` label in the password field

Image

[Figma](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=31-195&t=ulLcNTVal2MYSQPT-0)

### PR DESCRIPTION
**Pull request by phongvhd93**
- Close #19

## What happened 👀

- Create UI for the Login screen.
- Add font resources.
- Add view modifiers for `TextField` and `Button.`

## Insight 📝

When the user opens the application, we will show the login screen after the splash. To test this PR, change the line `KMMICApp:16` to `LogInView().`

Screenshot 2023-04-20 at 10 27 04

## Proof Of Work 📹

Screenshot 2023-04-20 at 10 19 53

### CODE REVIEW

File: iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryButton.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//  PrimaryButton.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 20/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct PrimaryButton: ViewModifier {
```
Comment (markgravity) [inline iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryButton.swift]: What do you think about renaming `PrimaryButton` to `PrimaryButtonModifier`? 
The current name is a bit confuse IMO 🙏

File: iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryTextField.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+//  PrimaryTextField.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 20/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct PrimaryTextField: ViewModifier {
```
Comment (markgravity) [inline iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryTextField.swift]: Same here 🙏

File: iosApp/PhongKMMIC/Sources/Supports/Extensions/SwiftUI/Font+Extension.swift [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+//  Font+Extension.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 20/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+extension Font {
```
Comment (markgravity) [inline iosApp/PhongKMMIC/Sources/Supports/Extensions/SwiftUI/Font+Extension.swift:11]: Nice 👍

**Review by markgravity — COMMENTED** [review]
The rest LGTM 👍

File: iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryButton.swift (hunk 1 above)
Comment (phongvhd93) [reply iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryButton.swift]: I solved it in 92d5442

File: iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryTextField.swift (hunk 2 above)
Comment (phongvhd93) [reply iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryTextField.swift]: I solved it in 92d5442

**Review by markgravity — APPROVED** [review]

**Review by suho — APPROVED** [review]

**Pull request merged**